### ISSUE TITLE
There are vulnerabilities related to the package nanoid

### ISSUE THREAD

**Issue by Anchit402** [postcss/postcss#1997]
latest version [8.4.49] postcss is using nanoid package with version 3.3.7 which has vulnerabilities
nanoid  <3.3.8
Severity: moderate
Predictable results in nanoid generation when given non-integer values - https://github.com/advisories/GHSA-mwcw-c2x4-8c55

**Comment by ai** [comment]
Yes, update Nano ID. We have `^` in `dependencies` which mean any version

**Comment by ai** [comment]
Please check closed issue next time before opening new issue.

You are saving your time by spending more time of free maintainer.
